opentelemetry-api/src/opentelemetry/attributes/__init__.py

```diff
@@ -121,12 +134,25 @@ def _clean_attribute(
 def _clean_extended_attribute_value(
     value: types.AnyValue, max_len: Optional[int]
 ) -> types.AnyValue:
+    # pylint: disable=too-many-branches
     # for primitive types just return the value and eventually shorten the string length
     if value is None or isinstance(value, _VALID_ATTR_VALUE_TYPES):
         if max_len is not None and isinstance(value, str):
             value = value[:max_len]
         return value
 
+    if _WSGIRequest is not None and isinstance(value, _WSGIRequest):
```

Comment:
I don't think this is the right place to handle this. This function expect `value` to be of type `AnyValue` so callers should respect that.